src/main/scala/com/github/unchama/seichiassist/subsystems/vote/domain/VotePersistence.scala

```diff
@@ -0,0 +1,57 @@
+package com.github.unchama.seichiassist.subsystems.vote.domain
+
+import java.util.UUID
+
+trait VotePersistence[F[_]] {
+
+  /**
+   * プレイヤーデータを作成する作用
+   */
```

Comment:
```suggestion
  /**
   * プレイヤーの投票データを初期化します。この作用はプレイヤーがサーバーに参加するたびに呼ばれるため、作用が冪等でなければなりません。
   * @return プレイヤーデータを作成する作用
   */
```

ドキュメントに加筆するならこうですが、これはSQLマイグレーションでやればいいのでは？

Reply:
このプレイヤーデータの作成は初めて参加したプレイヤーに対して行うことが目的なので、SQLマイグレーションだとちょっと違う気がします。